lib/kino/tree.ex

```diff
@@ -0,0 +1,139 @@
+defmodule Kino.Tree do
+  @moduledoc """
+  A kino for interactively viewing nested data as a tree view.
+
+  The data can be any term.
+
+  ## Examples
+
+      data = %{
+        id: 1,
+        email: "user@example.com",
+        inserted_at: ~U[2022-01-01T10:00:00Z],
+        addresses: [
+          %{
+            country: "pl",
+            city: "Kraków",
+            street: "Karmelicka",
+            zip: "00123"
+          }
+        ]
+      }
+
+      Kino.Tree.new(data)
+  """
+
+  use Kino.JS, assets_path: "lib/assets/tree"
+
+  def new(data) do
+    Kino.JS.new(__MODULE__, to_node(data))
```

Comment:
We can put this kino in a grid to get a bordered box as we do with inspect, wdyt?

<img width="911" alt="image" src="https://user-images.githubusercontent.com/17034772/202240155-d8c61f6b-54f3-4684-ad9f-c7c57f8c98f5.png">